ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -584,11 +584,12 @@ object Config {
           else success
         )
 
-        opt[Unit]("use-self-service-error-codes")
+        opt[Unit]("use-legacy-grpc-error-codes")
```

Comment:
Can we avoid using the word legacy, please? Why not simply have an option `disable-self-service-error-codes` or `enable-pre-1.18-error-codes`? Any of the latter would be more self-explanatory.